open_api_framework/conf/base.py

```diff
@@ -955,9 +956,7 @@ def init_sentry(before_send: Callable | None = None):
 # NOTE: make sure values are a tuple or list, and to quote special values like 'self'
 
 # ideally we'd use BASE_URI but it'd have to be lazy or cause issues
-CSP_DEFAULT_SRC = [
-    "'self'",
-] + config(
+CSP_DEFAULT_SRC = ["'self'", "'unsafe-inline'"] + config(
```

Comment:
Do we have to add `unsafe-inline` to the default? I'd prefer to add it only to the specific directives that need it (probably `script-src`)